src/Sarif/Writers/SarifOneZeroZeroLogger.cs

```diff
@@ -22,24 +22,29 @@ public class SarifOneZeroZeroLogger : SarifLogger
 
         public SarifOneZeroZeroLogger(
             string outputFilePath,
-            LoggingOptions loggingOptions = SarifLogger.DefaultLoggingOptions,
+            LogFilePersistenceOptions logFilePersistenceOptions = SarifLogger.DefaultLogFilePersistenceOptions,
             OptionallyEmittedData dataToInsert = OptionallyEmittedData.None,
             OptionallyEmittedData dataToRemove = OptionallyEmittedData.None,
             Tool tool = null,
             Run run = null,
             IEnumerable<string> analysisTargets = null,
             IEnumerable<string> invocationTokensToRedact = null,
             IEnumerable<string> invocationPropertiesToLog = null,
-            string defaultFileEncoding = null)
+            string defaultFileEncoding = null,
+            IEnumerable<FailureLevel> level = null,
```

Comment:
>level [](start = 38, length = 5)

nit: levels and kinds

Reply:
Done.

---
In reply to: [562033198](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r562033198) [](ancestors = 562033198)